src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1054,6 +1057,7 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(ngraph::pass::Ma
     pass_manager.register_pass<runtime::cpu::pass::CPUMemoryOptimization>();
     pass_manager.register_pass<ngraph::pass::GetOutputElementElimination>();
     pass_manager.get_state().set_visualize_tree_ops_map(runtime::cpu::get_visualize_tree_ops_map());
+    pass_manager.get_state().set_op_annotations_func(runtime::cpu::get_op_annotations_func());
```

Comment:
@ayzhuang @jbobba @diyessi  I was wondering if we could directly inject it into a pass instead of going through `State`like we did for CSE : https://github.com/NervanaSystems/ngraph/pull/1972/files#diff-a8b86c1cce292e0e2e89b028c5518ad6R1053 .There's a very good reason why `VisualizeTree` needs to go through state because we create more than one VisualizeTree pass dynamically in PassManager.